README.md

```diff
@@ -15,9 +15,21 @@ mvn clean package
 mvn spring-boot:run -Dspring.profiles.active=local
 ```
 
-## Deploy
+## Deployment
 
 ### Install
+
+1. Execute Onboarder install
+
+```
+cd partner-onboarder
+./install.sh
+```
+* During the execution of the 'install.sh' script, a prompt appears requesting information for the S3 bucket, including its name and URL.
```

Comment:
* During the execution of the `install.sh` script, a prompt appears requesting information for the S3 bucket, including its name and URL.